packages/remix-server-runtime/package.json

```diff
@@ -18,20 +18,15 @@
   "dependencies": {
     "@remix-run/router": "1.0.4",
     "@types/cookie": "^0.4.0",
+    "@types/react": "^18.0.15",
```

Comment:
We need this, because we're using `ComponentType` type in `packages/remix-server-runtime/routeModules.ts`.

We should probably move that to `@remix-run/react` and make `@remix-run/server-runtime` more renderer-agnostic in that place.

Reply:
It's fixed now, so no need to rush it 🤷‍♂️

Just wanted to notice you (and the team)